src/niscope/system_tests/test_system_niscope.py

```diff
@@ -125,3 +125,55 @@ def test_waveform_processing(session):
     session.clear_waveform_processing()
     session.horz_record_length == 4096
     session.horz_sample_rate == 10000000
+
+
+def test_configure_ref_levels(session):
+    session.configure_ref_levels()
+    session.commit()
+    assert session.meas_chan_high_ref_level == 90.0
+
+
+# revisit after #630 closed
```

Comment:
There's no way we'll ever find this. Do one of the following:

* Follow the convention and type this as TODO(lnjaleea): ... This is a string we search for.
* On the issue, make a note about revisiting this.

In this case, #630 is closed so just remove altogether.